### PR TITLE
GitHub configuration, Documentation and Website Cleanup

### PR DESCRIPTION
**Pull request by tuhaihe**
closes: #ISSUE_Number

---

## Change logs

This PR focuses on cleanup tasks for GitHub configuration, documentation, and website files, including the following updates:

1. GitHub Configuration

* Updated the project name and links in key GitHub configuration files, including: `.asf.yaml`, PR template, Workflows, `.gitmessage` and README.md.

2. Documentation Updates

* Updated Docusaurus configuration settings.
* Fixed broken anchor links in the documentation to reduce build warnings.

3. Website Changes

* Removed Chinese translation support for website pages (community/*, contribute/*, support/*, bootcamp/*), while preserving Chinese translations for documentation.
* Removed redundant files (coc.md and security.md) in favor of ASF policy documents.
* Updated outdated links and replaced old project names across various pages (Bootcamp, Community, Contribute, Support).

Note: While the documentation pages still support Chinese translations, website pages will now be English-only due to limited maintenance resources.

## Contributor's checklist

Here are some reminders before you submit your pull request:

* Make sure that your Pull Request has a clear title and commit message. You can take the [Git commit template](https://github.com/cloudberrydb/cloudberrydb/blob/main/.gitmessage) as a reference.
* Sign the Contributor License Agreement as prompted for your first-time contribution (*One-time setup*).
* Learn the [code contribution](https://cloudberry.apache.org/contribute/code) and [doc contribution](https://cloudberry.apache.org/contribute/doc) guides for better collaboration.
* Make sure that your changes deployment preview is successful.
* List your communications in the [GitHub Issues](https://github.com/cloudberrydb/cloudberrydb-site/issues) or [Discussions](https://github.com/apache/cloudberry/discussions) (if has or needed).
* Feel free to ask for the @cloudberrydb/doc team or other people to help review and approve.

### CODE REVIEW

File: src/consts/homeContent.tsx
Comment (edespino) [inline src/consts/homeContent.tsx:1]: This needs updating:
<LinkWithBaseUrl href="https://github.com/cloudberrydb/cloudberrydb">

File: src/consts/community.tsx
Comment (edespino) [inline src/consts/community.tsx:1]: There are `cloudberrydb` references that need to be updated.

**Review by edespino — CHANGES_REQUESTED** [review]
1. The Footer Security link generates a 404 because you removed the security.md.
2. The is also another Security link 404 on how-to-contribute page.
3. The output from npm run build is generating more warnings than it did in the past.
4. Shouldn't docusaurus catch these errors?
5. I used [linkcheck](https://github.com/filiph/linkcheck) to validate the site running locally. For your reference, attached you will find its output:
[linkcheck.txt](https://github.com/user-attachments/files/17947440/linkcheck.txt)

**Comment by tuhaihe** [comment]
> The output from npm run build is generating more warnings than it did in the past.

Yes, most of the issues occurred in the docs/ directory. I would like to collaborate with our documentation contributors to address them in a new PR for these warnings.

**Comment by tuhaihe** [comment]
Now running the `linkcheck` and returns:

```
▶ ~/Downloads/linkcheck/linkcheck --skip-file skip-urls.txt http://localhost:3000 --no-check-anchors
Perfect. Checked 63880 links, 382 destination URLs (1152 ignored). 
```

For the `missing anchor` warnings, have created one issue #188. Some of the anchors are right, but cannot be directed to the right place.

**Comment by tuhaihe** [comment]
> 1. The Footer Security link generates a 404 because you removed the security.md.
> 2. The is also another Security link 404 on how-to-contribute page.

They should be fixed.

**Review by edespino — APPROVED** [review]
+1 - Thank you.

**Pull request merged**